darts/tests/models/forecasting/test_local_forecasting_models.py

```diff
@@ -154,8 +155,9 @@ def test_save_load_model(self):
 
         for model in [ARIMA(1, 1, 1), LinearRegressionModel(lags=12)]:
             model_path_str = type(model).__name__
-            model_path_file = model_path_str + "_file"
-            model_paths = [model_path_str, model_path_file]
+            model_path_pathlike = pathlib.Path(model_path_str + "_pathlike")
```

Comment:
extending this test for invalid read/write path would be nice as well

Reply:
Done. I put it in an extra test though to avoid inflating the original test too much.